packages/runtime/test-runtime-utils/src/insecureTokenProvider.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ScopeType, ITokenClaims, IUser } from "@fluidframework/protocol-definitions";
+import { ITokenProvider, ITokenResponse } from "@fluidframework/routerlicious-driver";
+import { KJUR as jsrsasign } from "jsrsasign";
+
+/**
+ * As the name implies this is not secure and should not be used in production. It simply makes the example easier
+ * to get up and running.
+ */
+export class InsecureTokenProvider implements ITokenProvider {
```

Comment:
Nit: can we reuse the other token provider?

Reply:
Which one?